# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glibc/mips.patch.cross0
# Copyright (C) 2015 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- glibc-2.21/sysdeps/mips/preconfigure.vanilla	2015-02-10 12:57:19.200000000 +0000
+++ glibc-2.21/sysdeps/mips/preconfigure	2015-02-10 12:57:37.188000000 +0000
@@ -15,7 +15,8 @@
 	elif test "$abiflag" = "_ABI64" ; then
 		machine=mips/mips64/n64
 	else
-		as_fn_error $? "Unable to determine ABI." "$LINENO" 5
+		#as_fn_error $? "Unable to determine ABI." "$LINENO" 5
+		machine=mips/mips64/n64
 	fi
 	# $config_machine is not really needed here but the slash after
 	# $machine is needed by the case statement in
